index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+require_once './functions.php';
+$db = new PDO('mysql:host=db; dbname=kantopokemon', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare('SELECT * FROM `kanto pokemon plural`');
```

Comment:
does this table have spaces in the name? if yes you should change it to not have spaces, something like `kanto-pokemon` or `kantoPokemon`